corehq/apps/hqwebapp/static/hqwebapp/yaml/bootstrap5/requirejs.yml

```diff
@@ -12,7 +12,7 @@ fileExclusionRegExp: ^\.|\.css$
 paths:
   sentry_browser: "empty:"
   sentry_captureconsole: "empty:"
-  "tempus-dominus": "empty:"
+  "tempusDominus": "empty:"
```

Comment:
@orangejenny is this all I need to do (re your comment)?

Reply:
yes

Reply:
(and remove the es6 wrapper)

Reply:
The `:empty` config makes the es6 wrapper unnecessary. `:empty` just tells r.js not to parse tempus dominus for dependencies - which is fine, since it doesn't have any dependencies. So once you add `:empty`, it doesn't matter that tempus dominus uses the spread operator.

The ES6 wrapper is causing errors when I integrate it with the new build_requirejs changes (this PR: https://github.com/dimagi/commcare-hq/pull/34346). I'm not sure why yet. It might just be because the new build_requirejs changes get stricter about B5 js files containing B3 references.

Reply:
@orangejenny updated and should be good to go. thanks for the explanation! that makes sense